pkg/build/webhook/github/github_test.go

```diff
@@ -90,7 +90,7 @@ func TestWrongContentType(t *testing.T) {
 	client := &http.Client{}
 	req, _ := http.NewRequest("POST", server.URL+"/build100/secret101/github", nil)
 	req.Header.Add("Content-Type", "application/text")
-	req.Header.Add("X-Github-Event", "ping")
+	req.Header.Add("X-GitHub-Event", "ping")
```

Comment:
What?! Both forms are fine, I'm removing changes to X-Github-Event from this PR and we can make its usage consistent across the repo later. Afterall, this PR is about trigger names.